getting-started/chat-thread.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system chat is divided into threads. Threads are standalone parts of chat and can contain events. In every chat multiple users (customers or agents) can participate and every user can have multiple chats at the same time.
+
+New threads within chat are created on server side, algorithm of this is discussed below in *Advanced* section.
+
+![Chats and Threads](../images/chats.png "chats and threads")
+
+Events are some portion of data, that can be send to chat (in eg. messages), they are discussed in the next section of this guide.
+
+### Getting chats
+
+If you Log in to Agent, or Customer API you will receive on login response either chats_summary or last_chats_summary. Both of these objects contain some chat and thread ids, they are usefull to get your chatting history.
```

Comment:
If you Log in to Agent, or Customer API you will receive on login response either chats_summary or last_chats_summary. Both of these objects contain some chat and thread ids, they are usefull to get your chatting history. -> If you log in to the Agent or Customer API you will receive one of the following responses: chats_summary or last_chats_summary. Both of these objects contain some chat and thread IDs and can be used to retrieve the chat history.

Reply:
chodziło mi o to, że te idki mogą być przydatne do pobierania reszty historii, czy może być?:

If you log in to the Agent or Customer API you will receive one of the following in login response: chats_summary or last_chats_summary. Both of these objects contain some chat and thread IDs and these can be used to retrieve the chat history.